Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -1,96 +1,67 @@
-Enable Hubble
-==============
+Enable Hubble for Cluster-Wide Visibility
```

Comment:
in some sense hubble is already enabled. how about:
```suggestion
Enable Hubble Relay for Cluster-Wide Visibility
```

Reply:
I think Robin did this after discussing the doc changes with me. The title intentionally does not mention Hubble Relay, therefore let me defend this change :) 

From the perspective of someone who is trying out Cilium and Hubble for the first time, they should not worry about what components actually make Hubble work. We want users to associate "using Hubble" with "using Hubble Relay", as it is becoming the main entry point for the Hubble API and also a requirement for being able to use the UI (which I think is a much better first introduction to Hubble than the CLI).

While local mode is enabled by default, it's hard to use and does not come with the UI. While we don't need to hide this fact, it's probably not something we want people to worry about when using Hubble for the first time. So while "Enabling Hubble Relay" is absolutely the technically more correct here, from a users' perspective I think "Enabling Hubble" sends the better message.

Advanced users will have to read up on the more in-depth concepts section anyway to learn about how Hubble works internally and how to configure the different components individually.

Reply:
well defended 👍